lib/tasks/sam.rake

```diff
@@ -1,9 +1,9 @@
 namespace :sam do
   task check: :environment do
-    User.not_in_sam.each do |user|
+    User.where(sam_account: false).each do |user|
```

Comment:
That seems like a good idea. 

I took out the scopes because those can lead to a lot of pain. I have actually seen spreadsheets where the numbers don't add up because if you put scopes in different orders, different results. Also, if you think of scopes as an abstraction, you don't want to extract it until you have three usages.

Adding a patch on the SamAccountReckoner to hide what is actually happening seems harmless and easy to reason about!